src/test/java/com/conveyal/datatools/manager/jobs/NormalizeFieldTransformJobTest.java

```diff
@@ -0,0 +1,139 @@
+import org.apache.commons.lang3.ArrayUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Date;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import static com.conveyal.datatools.TestUtils.createFeedVersion;
+import static com.conveyal.datatools.TestUtils.zipFolderFiles;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class NormalizeFieldTransformJobTest extends DatatoolsTest {
+    private static final String TABLE_NAME = "routes";
+    private static final String FIELD_NAME = "route_long_name";
+    private static Project project;
+    private static FeedSource feedSource;
+    private FeedVersion targetVersion;
+
+
+    /**
+     * Initialize Data Tools and set up a simple feed source and project.
+     */
+    @BeforeAll
+    public static void setUp() throws IOException {
+        // start server if it isn't already running
+        DatatoolsTest.setUp();
+
+        // Create a project.
+        project = createProject();
+
+        // Create transform.
+        // In this test class, as an illustration, we replace "Route" with the "Rte" abbreviation in routes.txt.
+        FeedTransformation transformation = NormalizeFieldTransformation.create(
+            TABLE_NAME, FIELD_NAME, null, "Route => Rte");
+        FeedTransformRules transformRules = new FeedTransformRules(transformation);
+
+        // Create feed source with above transform.
+        feedSource = new FeedSource("Normalize Field Test Feed", project.id, FeedRetrievalMethod.MANUALLY_UPLOADED);
+        feedSource.deployable = false;
+        feedSource.transformRules.add(transformRules);
+        Persistence.feedSources.create(feedSource);
+    }
+
+    /**
+     * Clean up test database after tests finish.
+     */
+    @AfterAll
+    public static void tearDown() {
+        // Project delete cascades to feed sources.
+        project.delete();
+        feedSource.delete();
+    }
+
+    /**
+     * Run tear down after each test. This just deletes the feed versions that were used in the test.
+     */
+    @AfterEach
+    public void tearDownTest() {
+        // Clean up
+        if (targetVersion != null) targetVersion.delete();
+    }
+
+    @Test
+    public void canNormalizeField1() throws IOException {
+        canNormalizeField();
+    }
+
+    /**
+     * HACK: Second test used locally that checks that the correct ZIP file is picked up after being written.
+     * (This sort-of replicates the CI system running two builds in succession.)
+     */
+    @Test
+    public void canNormalizeField2() throws IOException {
+        if (!TestUtils.isCi()) {
+            canNormalizeField();
+        }
+    }
```

Comment:
I don't see how two CI builds running one-after-another would have any effect. And even if they did, whatever effect they have on each other should be logic that gets removed since tests should be able to be run independently (except perhaps for an e2e test).

Reply:
I removed the redundant test in https://github.com/ibi-group/datatools-server/pull/371/commits/e731ec47e8ade889bdc9cee2378c53955dfdc433 adding a note for the case of running that test on Windows machines.